app/javascript/components/Admin/UserAndProductsTabs.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link } from "@inertiajs/react";
+import React from "react";
+
+import { type User as UserType } from "$app/components/Admin/Users/User";
+import Tab from "$app/components/Tabs/Tab";
+import TabList from "$app/components/Tabs/TabList";
+
+type Props = {
+  selectedTab: string;
+  user: UserType;
+};
+
+const AdminUserAndProductsTabs = ({ selectedTab, user }: Props) => (
```

Comment:
Is this a separate component/file in order to be able to re-use this in affiliates? Asking because if so we probably want to be able to pass in the URLs rather than just the user, but I actually think this is simple enough it can just be inlined

Reply:
Can we move any more of the logic from `app/javascript/pages/Admin/Users/Products/Index.tsx` here? (looks like affiliates is essentially the exact same page with a couple of different routes, so can we do something like the user lists and just render a single shared component in the page?)

I'm fine to leave refactoring until we build affiliates though